Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -808,8 +808,8 @@ YUI.add('ez-platformuiapp', function (Y) {
                     view: "serverSideView",
                     callbacks: ['open', 'checkUser', 'handleSideViews', 'handleMainView']
                 }, {
-                    name: "adminRole",
-                    regex: /\/admin\/(pjax%2Frole.*)/,
+                    name: "adminLanguage",
```

Comment:
I guess it's a mistake, we need both routes if you want the roles and the language links in the admin navigation